lib/ret/media_search.ex

```diff
@@ -62,38 +63,89 @@ defmodule Ret.MediaSearch do
     favorites_search(cursor, type, account_id, q)
   end
 
+  def search(%Ret.MediaSearchQuery{source: "sketchfab", cursor: cursor, filter: nil, collection: nil, q: q}) do
+    if q == nil || q == "" do
+      search(%Ret.MediaSearchQuery{source: "sketchfab", cursor: cursor, filter: "featured", q: q})
+    else
+      query =
+        URI.encode_query(
+          type: :models,
+          downloadable: true,
+          count: @page_size,
+          max_face_count: @max_face_count,
+          max_filesizes: "gltf:#{@max_file_size_bytes}",
+          processing_status: :succeeded,
+          cursor: cursor,
+          q: q
+        )
+
+      sketchfab_search(query)
+    end
+  end
+
   def search(%Ret.MediaSearchQuery{source: "sketchfab", cursor: cursor, filter: "featured", q: q}) do
     query =
       URI.encode_query(
         type: :models,
         downloadable: true,
+        staffpicked: true,
         count: @page_size,
         max_face_count: @max_face_count,
+        max_filesizes: "gltf:#{@max_file_size_bytes}",
         processing_status: :succeeded,
-        sort_by: "-likeCount",
+        sort_by:
+          if q == nil || q == "" do
+            "-publishedAt"
+          else
+            nil
+          end,
         cursor: cursor,
         q: q
       )
 
     sketchfab_search(query)
   end
 
-  def search(%Ret.MediaSearchQuery{source: "sketchfab", cursor: cursor, filter: filter, q: q}) do
+  def search(%Ret.MediaSearchQuery{source: "sketchfab", cursor: cursor, filter: nil, collection: collection_id, q: q}) do
     query =
       URI.encode_query(
         type: :models,
         downloadable: true,
         count: @page_size,
         max_face_count: @max_face_count,
+        max_filesizes: "gltf:#{@max_file_size_bytes}",
         processing_status: :succeeded,
+        sort_by:
+          if q == nil || q == "" do
```

Comment:
if its a collection, should we care about the query here?

Reply:
Yeah, you can search within a collection, which can be useful for larger collections.

Reply:
sorry, what i meant is if the `sort_by` should be `nil` if we have a query. i guess that seems right assuming sketchfab will sort by relevance.

Reply:
Yeah, I'm pretty sure it does.